scripts/class/game.js

```diff
@@ -1 +1,27 @@
 // Class Game : gère la logique principale du jeu.
+
+class Game {
+  constructor(player, secretCode, numberTry) {
+    this.player = player;
+    this.secretCode = secretCode;
+    this.attemptCount = 0;
+    this.numberTry = numberTry;
+  }
+
+  isGameContinue() {
+    if (this.attemptCount > this.numberTry) {
```

Comment:
tu peux faire directement return this.attemptCount <= this.numberTry

Reply:
ok fait, dès que j'ai une méthode logique qui me renvoie un boléen, je penserais à faire un return direct